modules/devkit-support/src/test/java/org/mule/security/oauth/processor/DefaultHttpCallbackTestCase.java

```diff
@@ -40,7 +39,7 @@
 import org.slf4j.LoggerFactory;
 
 @RunWith(MockitoJUnitRunner.class)
-public class DefaultHttpCallbackTestCase extends AbstractMuleContextTestCase
+public class DefaultHttpCallbackTestCase extends FunctionalTestCase
```

Comment:
Having this test a functional means that something that used to work in an AbstractMuleContextTestCase does not work anymore. Revert this and fix it propertly